autoPyTorch/pipeline/components/setup/network_embedding/base_network_embedding.py

```diff
@@ -49,4 +49,4 @@ def build_embedding(self, num_input_features: np.ndarray, num_numerical_features
     #     # for i, category in enumerate(categories):
     #     #     num_input_features[num_numerical_columns + i, ] = len(category)
     #     # return num_numerical_columns, num_input_features
-    #     return None, None
+        return None, None
```

Comment:
@ravinkohli can take over this question since it was his commit. I would need to double-check the implementation again for that.

Reply:
hey, @nabenabe0928, As I previously explained in a previous PR, I'll fix this once our experiments are done and we have decided on our approach to preprocessing. @ArlindKadra has told us to have the preprocessing in the pipeline with hyperparameters. I can do this right now but it will hamper the current experiments we are doing so I have decided to wait.